account_invoice_pricelist/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Pricelist on Invoices
+=====================
+
+* Add a stored field pricelist on invoices, related to the partner pricelist;
+* Possibility to group by pricelist on account.invoice view;
+
+.. image:: static/src/description/screenshot_group_by.png
+
+
+This module doesn't add real feature by it self for end-users, but is usefull
+to do reporting, in a inherited module.
+
+Installation
+============
+
+Classical installation.
+
+Configuration
+=============
+
+Nothing to do.
+
+Note
+====
+
+* The field will not be stored for purchase invoices as long as 'purchase'
```

Comment:
The new pricelist_id field set in account invoice take the value of : 
- invoice.partner_id.property_product_pricelist.id, if it's a sale invoice; (defined in 'product' module)
- invoice.partner_id.property_product_pricelist_purchase.id, if it's a purchase invoice; (defined in 'purchase' module)
It's not symetric and I guess it's problem of Odoo core (purchase pricelist should be defined in product module) and I don't want to force people who want to use this module, to install purchase. That's why it's optional, with a warning in the log.
Regards.